apps/src/jigsaw/jigsaw.js

```diff
@@ -144,6 +144,7 @@ Jigsaw.init = function (config) {
 
   config.enableShowCode = false;
   config.enableShowBlockCount = false;
+  config.showUnusedBlocks = false;
```

Comment:
I'm not sure this is really needed, since Jigsaw levels have no "Run" button.  Can we remove this, and make `Blockly.showUnusedBlocks = true` the default everywhere?

Reply:
Ah I see, this is needed so unconnected Jigsaw blocks aren't partially transparent.